src/helpers/shareDialogs.ts

```diff
@@ -3,11 +3,10 @@ import { Share, ShareContent, ShareOptions, Platform } from 'react-native';
 import { Feed } from '../models/Feed';
 
 export const showShareFeedDialog = async (feed?: Feed) => {
-    const url = feed != null ? feed.url : '';
     const title = 'Share your channel';
-    const message = Platform.OS === 'android' ? url : undefined;
+    const url = feed != null ? feed.url : '';
+    const message = `Follow my channel on Felfele (https://felfele.org/app) by copying this message ${url} and tap the add channel button on the Home screen!`;
```

Comment:
If it's possible to put newlines in the text it would be better to structure it a bit differently:
```
Follow my channel on Felfele (https://felfele.org/app)
Copy this message and tap the add channel button on the Home screen!

${url}
```